auter

```diff
@@ -106,10 +107,22 @@ function prepare_updates() {
       # If check-update has an exit code of 100, updates are available.
       if [[ "${RC}" -eq 100 ]]; then
         sleep $(($RANDOM % ${MAXDELAY}))
-        PREPOUTPUT=$(${PACKAGE_MANAGER} ${PACKAGEMANAGEROPTIONS} update --downloadonly -y)
-        logit "INFO: Updates downloaded"
+        if [[ "${ONLYINSTALLFROMPREP}" == "yes" ]]; then
+          rm -f ${DOWNLOADDIR}/${CONFIGSET}/*.rpm
+          # DNF doesn't support downloaddir, so instead we download to the default
+          # location and copy out the way
+          if [[ "${PACKAGE_MANAGER}" == "dnf" ]]; then
+            find /var/cache/dnf -name *.rpm -exec rm -f {} \;
+            DOWNLOADOPTION=""
+          fi
+          DOWNLOADLOGMSG=" to ${DOWNLOADDIR}/${CONFIGSET}"
```

Comment:
This variable is used later on (line 123). If this variable is set then the logit message reads:
"INFO: Updates downloaded to /var/cache/auter/default"

If it is not set (ie when $ONLYINSTALLFROMPREP == no) it will read:
"INFO: Updates downloaded"

If I take out the " to" then we will end up with:
"INFO: Updates downloaded /var/cache/auter/default"

I am open to suggestions...